### PR TITLE
Increases the herd information minimum values to prevent issue  #1644

### LINKED ISSUE #1644: Change Minimum Number of Animals in Input Metadata
Currently the minimum number of each animal type is set to 0. However, if 0's are actually passed as inputs to some of the animal numbers, it causes a runtime error. We would like to change the minimum number so any inputs passsed to the simulation engine run correctly. 

## Expected Behavior

We would like the inputs that pass the IM's validation method to be those which allow for a successful simulation run so would like to change the . 
## Current Behavior

When 0's are passed to calf_num or any number less than 6 is passed to cow_num, a runtime error occurs. 

## Possible Solution

I would like to change the minimum number for the animal number inputs in the animal input json to be the following:

1. `"calf_num"`: 2
2. `"heiferI_num"`:2
3. `"heiferII_num"`:2
4. `"heiferIII_num_springers"`:2
5. `"cow_num"`: 6
6. `"replace_num"`: 50
7. `"herd_num"`: 6

## Steps to Reproduce

1. Run the default metadata on main with 0 calves or 0 cows as initial values. 
8.
9.
10.

## Context (Environment)

FARM-ES wants to make sure that the inputs they pass to the model don't cause it to crash

## Target Deadline

End of May

### PR DESCRIPTION
**Pull request by Fjordrunner**
This PR increases the minimum input values for certain herd information fields, such as `calf_num` and `cow_num`.

### Context
Issue(s) closed by this pull request: closes #1644 .

- [x] The [changelog](https://docs.google.com/spreadsheets/d/1U-3igxdstHTFb6k5dVcE5-x9IL0r6q18eUiwwytKZyA/edit#gid=0) has been updated.
      
How big are the changes in the PR? Would you nominate it for a major version upgrade?
- [ ] Yes
- [x] No

### What
This PR changes the minimum values for the herd information input fields of `calf_num`, `heiferI_num`, `heiferII_num`, `heiferIII_num_springers`, `cow_num`, and `replace_num` to 2, 2, 2, 2, 6, 50, and 6, respectively. These formerly had minimum values of 0.

### Why
Inputting values of 0 for some of these numbers resulted in various errors upon running the simulation, as discovered by FARM-ES. Additionally, having 0 of a certain herd animal type is rather unrealistic.

### How
This pull request altered the specified minimum values given in `input/metadata/properties/default.json`. The specific area tested is found under the JSON field `animal_properties` and sub-property `herd_information`.

### Test plan
We currently have various unit tests ensuring that if the model receives some input data that violates the minimum value requirements, the default value for the field(s) in question is used instead. These defaults are specified in `input/metadata/properties/default.json`. Several manual tests were also performed. If one wants to replicate any of them the following steps can be taken:

1. Within the RUFAS directory, navigate to the JSON file at the path `input/data/tasks/default_task.json`. Once there, change the "log_verbosity" field to have a string value of "warnings." This will change the model so that all of the warnings logged by the output manager are printed in the terminal.
2. Once again, within the RUFAS directory, navigate to the JSON file at the path `input/data/animal/default_animal.json`. Once there, set the value of the "calf_num" subfield within the "herd_information" field to some invalid value, such as 0, which is below the new minimum of 2 for that specific subfield. 
3. Now, within the RUFAS directory, navigate to`simulation_engine.py` within the model and navigate to line 178. Specifically, you can search for the following line: `animal_class_config = im.get_data("animal")`. This line creates the animal class configuration that the model will run on. Directly after this line, I suggest inserting the trace `print(animal_class_config["herd_information"])`. This will output the herd information that the model will run on, which has been processed by the input manager.
4. Now, run the model using `python main.py` in the terminal. One of the first outputted warnings should indicate that the value you inputted for the number of calves is below the minimum threshold of 2. Then, later on in the printed output, you will be able to see the herd information the model has opted to use. Here, we see that the animal configuration's herd information adopts the default number of calves(8), not the value we originally gave as input (0), has been used, and the simulation runs seamlessly.

### CODE REVIEW

File: input/metadata/properties/default.json [hunk 1]
```diff
@@ -44,43 +44,44 @@
...
+        "minimum": 6
       },
       "replace_num": {
         "type": "number",
         "description": "Replacements (head) -- Number of replacement animals available in the replacement market",
         "default": 5000,
-        "minimum": 0
+        "minimum": 50
+      },
+
```
Comment (ew3361zh) [inline input/metadata/properties/default.json]:
```suggestion
```

**Review by ew3361zh — APPROVED** [review]
LGTM!

**Review by Isgotkowitz — APPROVED** [review]
LGTM

**Comment by KFosterReed** [comment]
@Fjordrunner are you waiting on something for this PR? Our Farm ES partners would like confirmation that this is implemented in the main branch before they pull the next set of changes

File: input/metadata/properties/default.json (hunk 1 above)
Comment (Fjordrunner) [reply input/metadata/properties/default.json]: Just removed this extra line, thank you for for catching it!

**Comment by Fjordrunner** [comment]
> @Fjordrunner are you waiting on something for this PR? Our Farm ES partners would like confirmation that this is implemented in the main branch before they pull the next set of changes

We are good to go! I was running an additional manual test, sorry for the delay

**Pull request merged**